### ISSUE TITLE
Backup file not found (Mac)

### ISSUE THREAD

**Issue by SeanWhipple** [mykeels/slack-theme-cli#19]
When I execute the install commands it says that a backup file is not found.  It appears to be looking for /Users/<username>/.bash_profile

That file does exist and within it I can see
`export SLACK_THEME_SHELL_PROFILE="/Users/s.whipple/.bash_profile"` 

Is there something else I need to execute?

**Comment by mykeels** [comment]
Could you try re-installing? 

```
slack-theme uninstall && slack-theme install
```

**Comment by SeanWhipple** [comment]
I get the following
`C02T801JGTDY:~ s.whipple$ slack-theme uninstall && slack-theme install
sh: /Users/s.whipple/.bash_profile
uninstalled successfully
-bash: /Users/s.whipple/bin/slack-theme/slack-theme: No such file or directory`

**Comment by mykeels** [comment]
**face-palm**

Sorry, my bad ... if you've uninstalled, you won't have the `slack-theme` command anymore.

Run the installation script from the README instead.

**Comment by SeanWhipple** [comment]
The install appears successful 
```
C02T801JGTDY:~ s.whipple$ curl https://raw.githubusercontent.com/mykeels/slack-theme-cli/master/slack-theme -O && bash slack-theme install && . ~/.bash_profile
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  9349  100  9349    0     0  67357      0 --:--:-- --:--:-- --:--:-- 67746
sh: /Users/s.whipple/.bash_profile
sh: /Users/s.whipple/.bash_profile
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   888  100   888    0     0   6178      0 --:--:-- --:--:-- --:--:--  6209
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   104  100   104    0     0    727      0 --:--:-- --:--:-- --:--:--   727
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  206k  100  206k    0     0  1245k      0 --:--:-- --:--:-- --:--:-- 1250k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  206k  100  206k    0     0  1020k      0 --:--:-- --:--:-- --:--:-- 1021k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  207k  100  207k    0     0   733k      0 --:--:-- --:--:-- --:--:--  735k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  205k  100  205k    0     0  1142k      0 --:--:-- --:--:-- --:--:-- 1139k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  206k  100  206k    0     0  1116k      0 --:--:-- --:--:-- --:--:-- 1120k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  205k  100  205k    0     0   994k      0 --:--:-- --:--:-- --:--:--  997k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  206k  100  206k    0     0   955k      0 --:--:-- --:--:-- --:--:--  956k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  206k  100  206k    0     0  1197k      0 --:--:-- --:--:-- --:--:-- 1205k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  206k  100  206k    0     0  1189k      0 --:--:-- --:--:-- --:--:-- 1186k
installed successfully
```

But the command still fails :(
```
C02T801JGTDY:~ s.whipple$ slack-theme night
sh: /Users/s.whipple/.bash_profile
backup file not found
```

**Comment by no1joel** [comment]
I ran into this error; it happened for me because I'd previously installed a theme before installing slack-theme, so slack-theme never created the "clean" js file from which it creates the themed js file.
Inspecting the slack-theme script, the file that is changed is 
`/Applications/Slack.app/Contents/Resources/app.asar.unpacked/src/static/ssb-interop.js` on a Mac, or `/usr/lib/slack/resources/app.asar.unpacked/src/static/ssb-interop.js` on linux.

I went into this file, found the block at the bottom that contained `DOMContentLoaded` and removed it; this is the block that slack-theme uses to determine whether or not the file is currently modified. I re-ran slack-theme and all was well.
